core/invoker/src/main/scala/org/apache/openwhisk/core/containerpool/ContainerProxy.scala

```diff
@@ -429,32 +473,36 @@ class ContainerProxy(factory: (TransactionId,
     case Event(job: Run, data: WarmedData) =>
       implicit val transid = job.msg.transid
       activeCount += 1
-
-      initializeAndRun(data.container, job)
+      val newData = data.withResumeRun(job)
```

Comment:
Is this to always reschedule the job in case of ContainerHealthError no matter a container is resumed or not?